test/test_file_bundle.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import os
+import openminds.latest.core as omcore
+from openminds import Collection
+from bids2openminds.main import create_file_bundle
```

Comment:
if test is based on code above the assumed file bundle number is wrong (one less for the not needed dataset file bundle, and another (potentially) one less for the not needed tracking of the openminds folder which does not belong to the BIDS structure)